moped-editor/.env-cmdrc

```diff
@@ -91,12 +73,6 @@
     "REACT_APP_KNACK_DATA_TRACKER_PROJECT_VIEW": 3089,
     "REACT_APP_KNACK_DATA_TRACKER_SIGNAL_VIEW": 1483,
     "REACT_APP_KNACK_DATA_TRACKER_APP_ID": "621958d0b5ab96001edcb4b1",
-    "REACT_APP_KNACK_DATA_TRACKER_PROJECT_ID_FIELD": "field_4133",
-    "REACT_APP_KNACK_DATA_TRACKER_PROJECT_NAME_FIELD": "field_3857",
-    "REACT_APP_KNACK_DATA_TRACKER_CURRENT_STATUS_FIELD": "field_4136",
-    "REACT_APP_KNACK_DATA_TRACKER_SIGNAL_CONNECTION": "field_3861",
-    "REACT_APP_KNACK_DATA_TRACKER_MOPED_URL": "field_4161",
-    "REACT_APP_KNACK_DATA_TRACKER_MOPED_LINK_LABEL": "field_4162",
```

Comment:
we used to manage two URL fields - one was a text field and the other an object. i poked around in the knack apps and confirmed that the text field can be removed. we're not using it anymore. i deleted it from the and prod Data Tracker.